types/jscodeshift/src/testUtils.d.ts

```diff
@@ -19,3 +19,11 @@ export function defineInlineTest(
   expectedOutputSource: string,
   testName?: string
 ): () => any;
+
+export function runInlineTest(
+  module: Transform,
+  options: Options,
+  input: string,
```

Comment:
Is this a `string` or `{ path: string; input: string }` object?

Reply:
For example,

https://github.com/facebook/jscodeshift/blob/2fd5e11f469427d474983b2d1c47be9408677591/src/testUtils.js#L122-L124

https://github.com/facebook/jscodeshift/blob/2fd5e11f469427d474983b2d1c47be9408677591/src/testUtils.js#L97-L100